UniversalAdapter/src/androidTest/java/com/raizlabs/universaladapter/test/merged/MergedAdapterTest.java

```diff
@@ -44,4 +46,112 @@ public void testMergedAdapter() {
         }
 
     }
+
+    public void testMergedAdapterNotifications() {
+        MergedUniversalAdapter mergedUniversalAdapter = new MergedUniversalAdapter();
+
+        final int[] starts = new int[3];
+        final int[] counts = new int[3];
+        final boolean[] called = new boolean[4];
+        mergedUniversalAdapter.getListObserver().addListener(new ListObserverListener() {
+            @Override
+            public void onItemRangeChanged(ListObserver listObserver, int start, int count) {
+                starts[0] = start;
```

Comment:
#nth Might be easier to read if we broke these indices out as constants like `INDEX_CHANGE` `INDEX_INSERT`